proto/clustermetadata.proto

```diff
@@ -94,56 +93,19 @@ message MultiPooler {
   // If range based sharding is used, range for the pooler's shard.
   KeyRange key_range = 5;
 
-  // Type is the kind of pooler: PRIMARY or REPLICA
-  Type type = 6;
+  // PoolerType is the kind of pooler: PRIMARY or REPLICA
```

Comment:
We discussed this rename in previous PR, but missed the commit before it was merged 😅